### ISSUE TITLE
Request: rename Codec ID to File Type for 14496 family files

### ISSUE THREAD

**Issue by GyanD** [MediaArea/MediaInfoLib#978]
Mediainfo reports the major brand / compatible brands for ISO 14496 family files under the key of "Codec ID". Using the word 'Codec' here is asking for confusion, especially among novice or casual users. The spec itself calls the box containing this data as the file-type box.

Please considering changing the field name.

**Comment by JeromeMartinez** [comment]
"CodecID" is a generic field name for more than MP4, even if currently this is only for MP4 at the container level (it is used for identifying also video, audio, text...)
Which kind of confusion? 

If we change, it would not be for "file type", as it is IMO too much similar/synonym with "format", and too MP4-centric, I would look for another field name (which one?).

--> No definitive answer for the moment.

**Comment by Nadahar** [comment]
`type` might be generic and different enough to work..? Or does it conflict with something else?

**Comment by JeromeMartinez** [comment]
It could be something more like `Format flavor` (as it is related to the format i.e. a "sub-format"), but as said no decision for the moment about changing it so I don't focus of the new name for the moment.

**Comment by Nadahar** [comment]
Yeah, `type`, `variant`, `flavor` are all generic names that come to mind, and of course `sub-format` and `sub-type`. I see that it's not a priority, I just wanted to give some suggestions if/when they are needed :wink:

**Comment by GyanD** [comment]
_Which kind of confusion?_

That this field provides any insight about the identity of the bitstream syntax of the streams in the file.

For ftyp, `Format Flavor`. sounds good.
